DoctorWho.Console/ProgramToExecPreviousExercise.cs

```diff
@@ -0,0 +1,73 @@
+﻿using DoctorWho.Db;
+using DoctorWho.Db.DataModels;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System.Data;
+
+DoctorWhoCoreDbContext context = new DoctorWhoCoreDbContext();
+//printView();
+void printView()
+{
+    var command = "SELECT * FROM viewReport";
+    var result = context.viewReport.FromSqlRaw(command).ToList();
+    foreach (var e in result)
+        Console.WriteLine(e.ToString());
+}
+
+//GetEnemiesNameByID(301);
+void GetEnemiesNameByID(int id)
+{
+    var result = context.Episodes
+                  .Where(e => e.EpisodeId == id)
+                  .Select
+                  (a => new
+                  {
+                      EnemiesName = DoctorWhoCoreDbContext.GetEnemiesByEpisodeId(id)
+                  }
+                  ).ToList();

```

Comment:
is this the only way to do that? calling db function inside the context?

Reply:
I don't think it's a good way

Reply:
I will change it to non-static function.